tests/backend/test_apiv1_organization_stats.py

```diff
@@ -19,6 +21,7 @@ async def organization_stats(sock, organization_id):
 
 
 @pytest.mark.trio
+@customize_fixtures(adam_profile=UserProfile.OUTSIDER)
```

Comment:
I think we should split this `test_organization_stats` into:
- `test_organization_stats_data`: this test without the `customize_fixture` (given we don't care at all about users profiles here)
- `test_organization_stats_users`: a brand new test that creates 2 users of each type, check the stats, revoke user 1 user of each type, check the stats